README.md

```diff
@@ -0,0 +1,62 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
```

Comment:
- add Sentiment analysis
- automate moderation
- explain how admin will survive in logs
- add usecase scenarios to use case diagram
- read about data models of chat storage and improve ED diagram